test/jest/victory-voronoi/victory-voronoi.test.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * Client tests
+ */
+/*eslint-disable max-nested-callbacks */
+import React from "react";
+import { range, random } from "lodash";
+import { calculateD3Path } from "../../svg-test-helper";
+import { VictoryVoronoi, Voronoi } from "victory-voronoi";
+import { fireEvent, render } from "@testing-library/react";
+
+describe("components/victory-voronoi", () => {
+  describe("default component rendering", () => {
+    it("accepts user props", () => {
+      const { container } = render(
+        <VictoryVoronoi data-testid="victory-voronoi" aria-label="Chart" />
+      );
+
+      const svgNode = container.querySelector("svg");
+      expect(svgNode.getAttribute("data-testid")).toEqual("victory-voronoi");
+      expect(svgNode.getAttribute("aria-label")).toEqual("Chart");
+    });
+
+    it("renders an svg with the correct width and height", () => {
+      const { container } = render(<VictoryVoronoi />);
+      const svg = container.querySelector("svg");
+      expect(svg.style.width).toEqual("100%");
+      expect(svg.style.height).toEqual("100%");
+    });
+
+    it("renders an svg with the correct viewbox", () => {
+      const { container } = render(<VictoryVoronoi />);
+      const svg = container.querySelector("svg");
+      const viewBoxValue = `0 0 ${450} ${300}`;
+      expect(svg.getAttribute("viewBox")).toEqual(viewBoxValue);
+    });
+  });
+
+  describe("component rendering with data", () => {
+    it("renders the correct d3 path", () => {
+      const props = {
+        width: 400,
+        height: 300,
+        padding: 50,
+        domain: { x: [0, 5], y: [0, 5] },
+        data: [
+          { x: 0, y: 0 },
+          { x: 2, y: 3 },
+          { x: 4, y: 1 }
+        ]
+      };
+      const { container } = render(<VictoryVoronoi {...props} />);
+
+      expect(container.querySelector("path").getAttribute("d")).toEqual(
+        calculateD3Path(props, "voronoi", 0)
+      );
+    });
+
+    it.skip("sorts data by sortKey prop", () => {
```

Comment:
This test is a weird one to convert. Because most Victory props can be accessor functions that can get prop values, I have a really hacky solution here that accesses the data prop and stringifies it in the DOM for RTL to access. https://github.com/FormidableLabs/victory/blob/5f60e9a700b9358a47792f3d137ef497abe71aee/test/jest/victory-line/victory-line.test.js#L162

However, this also might be a case for testing this functionality in a different place so we aren't using weird workarounds like this to access the props being passed into the individual components.

Reply:
I got the weird workaround to work, and was able to re-enable these skipped tests.  But I agree, we should consider improving this approach.